Loop/Managers/LoopDataManager.swift

```diff
@@ -126,6 +126,19 @@ final class LoopDataManager {
 
                     self.insulinEffect = nil
 
+                    self.notify(forChange: .bolus)
+                }
+            },
+            NotificationCenter.default.addObserver(
```

Comment:
The `addObserver` above this is not specifying a name for notifications from doseStore. Could you add the right name? I believe it is DoseStore.valuesDidChange.

Reply:
Yep I’ll probably be able to do that tomorrow night or this weekend.